src/assets.rs

```diff
@@ -126,11 +126,12 @@ impl HighlightingAssets {
         Ok(())
     }
 
-    pub fn default_theme(&self) -> Result<&Theme> {
+    pub fn get_theme(&self, theme: &str) -> Result<&Theme> {
+        let err = format!("Could not find '{}' theme", theme);
```

Comment:
I think this should be inlined in the `ok_or_else` call. Otherwise, it will always be constructed.

Reply:
Agreed.